docs/services/README.md

```diff
@@ -5,7 +5,7 @@
 |            	| Catalog Application 	| `<RASTER-CATALOG-SERVICE_URL>`        	|
 | **Raster** 	|                     	|                                       	|
 |            	| Catalog             	| `<RASTER-CATALOG-SERVICE_URL>`        	|
-|            	| Map                 	| `<RASTER-RASTER-SERVING-SERVICE_URL>` 	|
+|            	| Map                 	| `<RASTER-SERVING-SERVICE_URL>` 	        |
 |            	| Export              	| `<RASTER-EXPORT-SERVICE_URL>`         	|
 | **Vector** 	|                     	|                                       	|
 |            	| Replication         	| `<VECTOR-REPLICATION-SERVICE_URL>`    	|
```

Comment:
I disagree with the change. I want to keep it as RASTER-RASTER, I see no problem with that

Reply:
reverted